pkg/pow/miningTasker.go

```diff
@@ -132,13 +136,29 @@ func (mt *MiningTasker) GetWork(chan *Work) (*Work, bool) {
 		m2, err := mt.proxy.BatchGet([]string{valKey})
 		if err != nil {
 			mt.log.Info("Could not retrieve pricing data for current request id: %v", err)
-			return nil, false
+			//return nil, false
 		}
 		val := m2[valKey]
-		if len(val) == 0 {
+		if val == nil || len(val) == 0 {
+			cfg := config.GetConfig()
+			indexPath := filepath.Join(cfg.ConfigFolder, "manualData.json")
```

Comment:
But the manual data is already added with the `BuildIndexTrackers` and `configs/indexes.json`

https://github.com/tellor-io/TellorMiner/blob/e97654525c2d34f45979e045311ec9cdf7cf23ad/configs/indexes.json#L147

Since https://github.com/tellor-io/TellorMiner/issues/245 points out that 0 values should not be submitted I think here it should just return an error. The same inside the solution handler.

Reply:
had a sync call with @themandalore and the problem is that the ovveride works on per symbol basis and not per request ID